umh-core/.golangci.yaml

```
@@ -0,0 +1,8 @@
+
```

YAMLlint (1.35.1)

[warning] too many blank lines

(1 > 0) (empty-lines)

---

[error] no new line character at the end of file

(new-line-at-end-of-file)